frontend/ui/widget/keyboardlayoutdialog.lua

```diff
@@ -58,15 +58,26 @@ function KeyboardLayoutDialog:init()
 
     local buttons = {}
     local radio_buttons = {}
+
+    local keyboard_layouts = G_reader_settings:readSetting("keyboard_layouts") or {}
+    G_reader_settings:makeTrue("keyboard_force_current_layout")
     for k, _ in FFIUtil.orderedPairs(self.parent.keyboard.lang_to_keyboard_layout) do
+        local text = Language:getLanguageName(k)
+        if keyboard_layouts[k] == true then
+            text = "✓ " .. text
```

Comment:
Looks a bit odd on your screenshot, making the language names no longer aligned.
What if on the right ?
(More complex and dunno how it would look would be having them bold.)

Reply:
<kbd>![5](https://user-images.githubusercontent.com/62179190/131211058-d8df85ab-6687-4234-b34d-f668598e4900.png)</kbd>

Reply:
^ Looks better to me.

Reply:
Done